tests/gtfs/test_unmatched_id_warnings.py

```diff
@@ -0,0 +1,152 @@
+"""Tests for inserting invalid data into various GTFS tables."""
+import pytest
+import pandas as pd
+import numpy as np
+
+from transport_performance.gtfs.validation import GtfsInstance
+
+
+@pytest.fixture(scope="function")
+def gtfs_fixture():
+    """Test GtfsInstance() fixture."""
+    gtfs = GtfsInstance()
+    return gtfs
+
+
+@pytest.mark.runinteg
```

Comment:
runinteg marked at class or unit test? We have both in this file. Probably best to have it at the test level.